alvr/dashboard/src/steamvr_launcher/mod.rs

```diff
@@ -112,6 +113,12 @@ pub struct Launcher {
 
 impl Launcher {
     pub fn launch_steamvr(&self) {
+        // The ADB server might be left running because of a unclean termination of SteamVR
+        // Note that this will also kill a system wide ADB server not started by ALVR
+        if let Some(path) = adb::get_adb_path(&crate::get_filesystem_layout()) {
+            adb::kill_server(&path).ok();
+        }
```

Comment:
This is alright but it would unnecessarily kill a global adb server instance. Better would be to get only the local adb path and try to kill that. But better yet would be to get the current running adb processes and kill any that is child of SteamVR (vrserver). This last solution would also be resistant to edge cases where the users installs a global adb server while running SteamVR

Reply:
note though, i think you can't run two adb instances as well, so you kinda have to replace one instance with another anyway